syntaxes/vba.tmLanguage.json

```diff
@@ -49,6 +49,10 @@
 					"name": "constant.language.vba",
 					"match": "(?i:\\b(Empty|False|Nothing|Null|True)\\b)"
 				},
+				{
+					"name": "constant.numeric.date",
+					"match": "#(?i:(A(pr|ug)|Dec|Feb|J(an|u[ln])|Ma(r|y)|Nov|Oct|Sep)|[0-9]{1,2})/[0-9]{1,2}(/[0-9]{1,4})?#"
```

Comment:
Tip: [February 29](https://www.baeldung.com/java-date-regular-expressions#3-matching-the-february-29th)

Reply:
This can be done later. Amount of improvements offsets it.